src/modules/SX127x.h

```diff
@@ -850,9 +860,12 @@ class SX127x: public PhysicalLayer {
     int16_t configFSK();
     int16_t getActiveModem();
     int16_t directMode();
+    
 
   private:
     float _dataRate;
+    size_t _packetLength;
+    bool _packetLengthQueried; //internal flag so SX/HopeRF radios don't lose data on repeat queries.
```

Comment:
Change comment to "FSK packet length is the first byte in FIFO, length can only be queried once"